docs/contribute/code.md

```diff
@@ -105,11 +114,13 @@ Unit tests ensure that Storybook doesn't break accidentally. If your code can re
 
 ### End-to-end tests (e2e)
 
-Storybook's monorepo is set up to rely on end-to-end testing with [Cypress](https://www.cypress.io/) during CI. To help with testing, we encourage running this test suite before submitting your contribution. Detailed below are some steps you can take:
+Storybook's monorepo is set up to rely on end-to-end testing with [Playwright](https://playwright.dev) during CI. To help with testing, we encourage running this test suite before submitting your contribution. 
 
-1. Ensure you have Storybook successfully built in your local branch (i.e., run `yarn bootstrap --core`)
-2. Open a terminal and run `yarn local-registry --port 6001 --open --publish` to publish Storybook's packages into a local registry.
-3. In a second terminal, set up a reproduction using the local registry and run the Cypress tests with `yarn test:e2e-framework`.
+To run a e2e test against a sandbox, you can use the `e2e-tests` task:
+
+```shell
+yarn task --task e2e-tests --template=<template> --start-from=auto
+```
```

Comment:
Might be possible that when people read this they might not know what `template` to fill in. Maybe specify `react-vite/default-ts` as an example, given it's the one coming from `yarn start`?

Reply:
Feature request: It would be nice if we could run the commands without specifying a template and it would prompt. For instance, running `yarn task --task e2e-tests --start-from=auto` without `--template` results in the following error:
<img width="657" alt="image" src="https://user-images.githubusercontent.com/1671563/194706806-0da496d8-29f0-4df4-9555-e96ee60d3b38.png">

Reply:
Done!